openshift/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# This will produce and image to be used in OpenShift
+# Build should be triggered from the repo root like:
```

Comment:
Any reason you're using "docker build" instead of creating a BuildConfig in OpenShift?